src/Npc_VisualWeapon.cpp

```diff
@@ -115,8 +115,8 @@ class npc_visualweapon : public CreatureScript
             itemTemplate->SubClass == ITEM_SUBCLASS_WEAPON_FISHING_POLE)
             return;
 
-        player->SetUInt16Value(PLAYER_VISIBLE_ITEM_1_ENCHANTMENT + (item->GetSlot() * 2), 0, visual);
-        CharacterDatabase.PExecute("REPLACE into custom_item_enchant_visuals VALUES ('%u', '%u', '%s')", item->GetGUIDLow(), visual, player->GetName().c_str());
+        player->SetUInt16Value(PLAYER_VISIBLE_ITEM_1_ENCHANTMENT + (item->GetSlot() * 2), 0, visual_id);
+        CharacterDatabase.PExecute("REPLACE into `mod_weapon_visual__enchantments` VALUES ('%u', '%u', '%s')", item->GetGUIDLow(), visual_id);
```

Comment:
Doube `_`

Reply:
i thought i had changed that -__- damnit